Rebuild RHEL facts on Vagrant

yakatz opened the pull request:
Fixes #276

yakatz commented on facts/3.14/redhat-7-x86_64.facts:
```diff
@@ -72,58 +72,71 @@
-  "interfaces": "eth0,eth1,lo",
+  "interfaces": "eth0,lo",
   "ipaddress": "10.0.2.15",
-  "ipaddress6": "fe80::5054:ff:fe8a:fee6",
```
I ran Vagrant to generate this. Do you know if there is a way to make sure Vagrant has IPv6?

yakatz replied:
Looks like a few more to add

bastelfreak commented on facts/3.14/redhat-7-x86_64.facts:
```diff
@@ -210,30 +297,39 @@
   "netmask6": "ffff:ffff:ffff:ffff::",
   "netmask6_eth0": "ffff:ffff:ffff:ffff::",
-  "netmask6_eth1": "ffff:ffff:ffff:ffff::",
-  "netmask6_lo": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff",
```
this needs to stay as well

yakatz replied:
Should there be multiple network interfaces? I can enable IPv6 for `lo`, but there is no `eth1`

bastelfreak commented on facts/3.14/redhat-7-x86_64.facts:
```diff
@@ -210,30 +297,39 @@
   "network6": "fe80::",
   "network6_eth0": "fe80::",
-  "network6_eth1": "fe80::",
-  "network6_lo": "::1",
```
same here

bastelfreak replied:
I don't see any facts that were missing previously and that you added, so maybe don't edit this factset at all?

zilchms commented:
@yakatz how is the status on this? :) stumbled upon errors in a different module due to the inconsistencies you address. any plans on addressing the feedback bastelfreak gave you and getting this PR merged? :)

yakatz commented:
> any plans on addressing the feedback bastelfreak gave you and getting this PR merged?

If course I have plans :-)
On the other hand, I have a deadline coming up to get my PhD dissertation done and I shouldn't be working on anything but that right now

zilchms commented:
Dissertation definitely has priority :) good luck on that

yakatz commented:
Rather than try to rebase this, I am closing this and starting again.
